ocaml-lsp-server/test/e2e/__tests__/__snapshots__/workspace-symbol.test.ts.snap

```diff
@@ -0,0 +1,1485 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`workspace/symbol returns symbols from workspace 1`] = `
+Array [
+  Object {
+    "kind": 2,
+    "location": Object {
+      "range": Object {
+        "end": Object {
+          "character": 23,
+          "line": 1,
+        },
+        "start": Object {
+          "character": 0,
+          "line": 1,
+        },
+      },
+      "uri": "file://<testWorkspacePath>/bin/.main.eobjs/dune__exe.ml-gen",
```

Comment:
These are the modules created from files. Do we want to keep them in the result? If so, we'll have to figure out how to get the correct `uri`

![image](https://user-images.githubusercontent.com/5595092/116771187-bde26f00-aa49-11eb-90df-19c1a7fd54ef.png)

Reply:
I think any cmt file with `ml-gen` as the source extension should be excluded.

Reply:
I went for keeping `.ml(i)` files only (a682d739311a399c1b6313c284927cc8fd15aac8). Does that make sense?

Reply:
Seems fine to me.